libs/utils/storage-keys.ts

```diff
@@ -0,0 +1,8 @@
+export type StorageKey = typeof StorageKey[keyof typeof StorageKey];
+
+export const StorageKey = {
+    ID_TOKEN: 'id_token',
+    ACCESS_TOKEN: 'access_token',
+    REFRESH_TOKEN: 'refresh_token',
+    EMAIL: 'Email'
```

Comment:
```suggestion
    EMAIL: 'Email'
```

Not sure why you need it